src/main/java/parkingLot/User.java

```diff
@@ -1,4 +1,5 @@
 package parkingLot;
 
 public interface User {
+  /**/
```

Comment:
instead of an empty comment, add TODO so we will get a list of all uncompleted interfaces, classes or methods.
```suggestion
  // TODO: complete
```